civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -6768,11 +6768,14 @@ function civihr_employee_portal_preprocess_page(&$vars) {
     $display_name = t('Anonymus');
     $edit_account_link = '';
     $logout_link = '';
+    $user_guide_link = '';
```

Comment:
I don't think the $user_guide_link depends on whether the session is set or not. It should always be the same, why not just declare it here?